operators/src/source/ogr_source.rs

```diff
@@ -533,8 +538,17 @@ where
             }),
             OgrSourceTimeFormat::Custom { custom_format } => Box::new(move |field: FieldValue| {
                 let date = field.into_string().ok_or(Error::OgrFieldValueIsNotString)?;
-                let date_time = DateTime::parse_from_str(&date, &custom_format)?;
-                Ok(date_time.timestamp_millis().try_into()?)
+                let date_time_result = DateTime::parse_from_str(&date, &custom_format)
```

Comment:
- can't you pull out the `timestamp_millis` because it's common among all three cases?
- can you please add a test for each case?

Reply:
i don't think so. its not the same type